src/timeMachine/components/Vis.tsx

```diff
@@ -25,12 +25,15 @@ import {
   getYSeriesColumns,
 } from 'src/timeMachine/selectors'
 import {getTimeRangeWithTimezone} from 'src/dashboards/selectors'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
+import {getColorMappingObjects} from 'src/visualization/utils/colorMappingUtils'
 
 // Types
 import {RemoteDataState, AppState, ViewProperties} from 'src/types'
 
 // Selectors
 import {getActiveTimeRange} from 'src/timeMachine/selectors/index'
+import {setViewProperties} from '../actions'
```

Comment:
minor nit:

```suggestion
import {setViewProperties} from 'src/timeMachine/actions'
```

inconsistencies, even minor ones, tend to draw attention to themselves, and someone reading the code might assume there is more meaning here than there should be.